docs/flow/reference/cli/fmt.md

```diff
@@ -0,0 +1,27 @@
+---
+aliases:
+- /docs/agent/latest/flow/reference/cli/run
```

Comment:
Alias looks wrong should it be `/docs/agent/latest/flow/reference/cli/fmt`

Reply:
🤦 my bad, fixed